starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="./css/main.css">
   <title>Apple Pie Recipe</title>
 </head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+<body>
+  <header>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+    <h1>Apple Pie</h1>
+  </header>
+  <section>
+    <p>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always be my favorite
+      and has won me several first place prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+    <img class="image" src=".\images\recipe-info.png" alt="recipe info">
+  </section>
+  <hr width="75%">
+  <section>
+    <h2>Ingredients</h2>
+    <ul>
+      <li>1 recipe pastry for a 9 inch double crust pie</li>
+      <li>1/2 cup unsalted butter</li>
+      <li>3 tablespoons all-purpose flour</li>
+      <li>1/4 cup water</li>
+      <li>3 tablespoons all-purpose flour</li>
+      <li>1/4 cup water</li>
+      <li>1/2 cup white sugar</li>
+      <li>1/2 cup packed brown sugar</li>
+      <li>8 Granny Smith apples - peeled, cored and sliced</li>
+    </ul>
+    <img class="image" src=".\images\cooking-info.png" alt="cooking info">
+  </section>
+  <hr width="75%">
```

Comment:
that's the only point where I can say something, you should keep the styling in the css (even if it's work like this) because on a big project you may loose yourself if you add piece of styling everywhere, better to have all in the same place :)